pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go

```diff
@@ -36,6 +37,10 @@ import (
 	k8s "sigs.k8s.io/vsphere-csi-driver/v2/pkg/kubernetes"
 )
 
+// EnvCSINamespace represents the environment variable which
+// stores the namespace in which the CSI driver is running.
+const EnvCSINamespace = "CSI_NAMESPACE"
```

Comment:
Can we change `CSI_NAMESPACE` to `vSphere_CSI_Namespace`

Reply:
drop this comment, as previously we used CSI_NAMESPACE at https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.3.0-rc.1/manifests/vanilla/vsphere-csi-driver.yaml#L180-L181